demo/vl-map-wms-layer.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>vl-map-wms-layer voorbeeld</title>
+  <script type="module" src="/node_modules/vl-ui-demo/dist/vl-demo-all.js"></script>
+  <script type="module" src="/node_modules/vl-ui-body/dist/vl-body.js"></script>
+
+  <script type="module" src="/src/vl-map-all.js"></script>
+
+  <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-body/dist/style.css"/>
+  <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-demo/dist/style.css"/>
+</head>
+
+<body is="vl-body">
+<vl-demo-page data-vl-webcomponent="vl-map-wms-layer">
+  <vl-demo data-vl-title="Kaart met WMS kaartlaag">
+    <vl-map id="map-with-wms-layer">
+      <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+      <vl-map-wms-layer data-vl-name="Beschermingszones"
+                        data-vl-styles=""
+                        data-vl-version="1.3.0"
```

Comment:
das de "wms" versie dan? misschien beter vl-wms-version dan?

Reply:
ja maar ik ging er van uit dat dat logisch was, we zeggen toch ook niet wms-url?

Reply:
nee das waar, maar ik ik had hier wel iets van "versie van wat" ...